website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -160,9 +172,15 @@
               >
                 <!-- @TODO: Why do we not sync quest
                   progress on the group doc? Each user could have different progress.-->
-                <span
-                  class="float-right"
-                >{{ (user.party.quest.progress.up || 0) | floor(10) }} {{ $t('pendingDamageLabel') }}</span> <!-- eslint-disable-line max-len -->
+                <span class="float-right">
+                  {{
+                    (Math.floor((user.party.quest.progress.up || 0) * 10) / 10)
+                      .toLocaleString(user.preferences.language, {
+                        minimumFractionDigits: 1,
+                        maximumFractionDigits: 1
+                      })
+                  }} {{ $t('pendingDamageLabel') }}
+                </span> <!-- eslint-disable-line max-len -->
```

Comment:
Line is no longer longer than max-len.
```suggestion
                </span>
```

Reply:
No idea how eslint works, kept it in.